amenity-points.mss

```diff
@@ -2565,10 +2593,17 @@
       [feature = 'leisure_ice_rink'],
       [feature = 'leisure_pitch'] {
         text-fill: darken(@pitch, 40%);
+        [sport = 'baseball'],
```

Comment:
If you add the SQL query as suggested above, you can change this to `[sport != null]` instead of writing up a long list, I think.

Reply:
I agree, it should be safe enough to add generic text-dy, so please change it.